package.json

```
@@ -22,7 +22,7 @@
     "packages/insomnia-scripting-environment"
   ],
   "scripts": {
-    "dev": "npm start -w insomnia",
+    "dev": "dotenv -- npm start -w insomnia",
```

```suggestion
    "dev": "npx -y dotenv-cli npm start -w insomnia",
```
In order to avoid adding a new dependency to maintain. 

Another idea would be to use `npx -y dotenv-cli npm run dev` in your own terminal, until you have a documented dx and sample .env files to propose, which will help justify making alterations to the dx.

---

👌 Good point.